js-opentracing/README.md

```diff
@@ -0,0 +1,48 @@
+This example shows you how to use [OpenTelemetry](https://opentelemetry.io) [OpenTracing](https://opentracing.io/)
+
+## Example
+
+```javascript
+const opentelemetry = require('@opentelemetry/api');
+const { NodeTracerProvider } = require('@opentelemetry/node');
+const { SimpleSpanProcessor, ConsoleSpanExporter } = require('@opentelemetry/tracing');
+const { CollectorTraceExporter } = require('@opentelemetry/exporter-collector');
+const tracerProvider = new NodeTracerProvider();
+const axios = require('axios');
+
+// TraceShim is needed to join OpenTelemetry with OpenTracing
+const { TracerShim } = require('@opentelemetry/shim-opentracing');
+const opentracing = require('opentracing');
+
+tracerProvider.addSpanProcessor(new SimpleSpanProcessor(new ConsoleSpanExporter()));
+tracerProvider.addSpanProcessor(new SimpleSpanProcessor(new CollectorTraceExporter({
```

Comment:
is it possible to do this with the launcher?

Reply:
done